pages/nfts/[address]/offers/new.tsx

```diff
@@ -0,0 +1,102 @@
+import { GetServerSideProps } from 'next';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { LoadingContainer } from '@/components/elements/LoadingPlaceholders';
+import Modal from '@/components/elements/Modal';
+import OfferForm from '@/components/forms/OfferForm';
+import BlurPage from '@/components/layouts/BlurPage';
+import { imgOpt } from '@/common/utils';
+import { useNftMarketplaceQuery, useNftPageQuery } from '../../../../src/graphql/indexerTypes';
+import { Nft, Marketplace } from '@/types/types';
+
+import Custom404 from '../../../404';
+import NftByAddress, { Avatar, OverlappingCircles } from '../../[address]';
+import { HOLAPLEX_MARKETPLACE_SUBDOMAIN } from '@/common/constants/marketplace';
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  return {
+    props: {
+      address: context?.params?.address ?? '',
+    },
+  };
+};
+
+const NewNFTOffer = ({ address }: { address: string }) => {
+  const [imgLoaded, setImgLoaded] = useState(false);
+
+  const { data, loading, called, refetch } = useNftMarketplaceQuery({
+    fetchPolicy: `no-cache`,
+    variables: {
+      subdomain: HOLAPLEX_MARKETPLACE_SUBDOMAIN,
+      address: address,
+    },
+  });
+
+  const nft = data?.nft;
+  const marketplace = data?.marketplace;
+
+  const router = useRouter();
+
+  const goBack = () => {
+    router.back();
+  };
+
+  if (called && !data?.nft && !loading) {
+    return <Custom404 />;
+  }
+
+  return (
+    <>
+      <BlurPage>
+        <NftByAddress address={address} />
+      </BlurPage>
```

Comment:
nb: I think layout still worth a review. https://nextjs.org/docs/basic-features/layouts#per-page-layouts

Reply:
I suggest we go through the app after this MVP and define a few layouts we can implement like
- discovery
- profiles
- details

probably will need a few nested layouts as well.

For now, as far as I can "see with my eyes" React does not do undue renders if the Wrapper components are the same between pages so I don't think it's critical